tmh_registry/registry/tests/api/viewsets/test_patients.py

```diff
@@ -100,6 +108,15 @@ def test_get_patients_detail_successful(self):
         self.assertEqual(HTTP_200_OK, response.status_code)
         self.assertEqual(self.patient.id, response.data["id"])
 
+    def test_get_patients_detail__with_episode_successful(self):
```

Comment:
minor,
rename `test_get_patients_detail__with_episode_successful`
in to `test_get_patients_detail_with_episode_successful`